examples/with-google-tag-manager/components/GoogleTagManager.js

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from 'react'
+import { useRouter } from 'next/router'
+import * as gtm from '../lib/gtm'
+
+const handleRouteChange = () => {
+  gtm.pageview()
```

Comment:
url should be passed to this method call so it sets a page view correctly

Reply:
From the [next/router docs](https://nextjs.org/docs/api-reference/next/router#usage-6), it looks like url is passed to `handleRouteChange`